packages/components-blog/package.json

```
@@ -18,6 +18,7 @@
   },
   "peerDependencies": {
     "astro": "^4.0.0",
-    "@gemini-ui-astro/components": "*"
+    "@gemini-ui-astro/components": "*",
+    "sass": "^1.69.7"
```

```suggestion
    "sass": "*"
```

Do we depend on a specific version of `sass`?

---

Was based on the version we have in the root package.json, will update 😄